File: src/3110.score-of-a-string.3379/solution.ts
function scoreOfString (s: string): number {
    let result = 0;
    for (let i = 0; i < s.length - 1; i++) {
        result += Math.abs(s.charCodeAt(i) - s.charCodeAt(i + 1));
    }
    return result;
}

File: src/3110.score-of-a-string.3379/solution.java
class Solution {
    public int scoreOfString(String s) {
        char[] chars = s.toCharArray();
        int result = 0;
        for(int i=0;i<chars.length-1;i++){
            result += Math.abs(chars[i]-chars[i+1]);
        }
        return result;
    }
}